setup.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
+
+# Installing poetry
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+# Add poetry to PATH in script
+export POETRY_ROOT=\"$HOME//.poetry/\"
+export PATH=\"$POETRY_ROOT/bin:$PATH\"
+
+#Checking if poetry is already installed
+poetry --version 
+
+#Install Python
+sudo apt-get update -y && sudo apt-get install -y python3.8 
+
+#Store local variables 
+cp .env.template .env 
+
+# Run the application
+poetry run flask run 
```

Comment:
Worth noting that on a fresh setup, this is likely to fail because although we've created a `.env` file, it won't by default contain any of your secrets. Whether we include this step as part of the setup might depend on whether our app depends on those secrets, we could reasonably just allow the user to run the app separately